ansible/group_vars/kafka.yml

```diff
@@ -0,0 +1,4 @@
+datadog_integrations:
+  - kafka
+  - kafka_consumer
+  - zk
```

Comment:
for now we always deploy zk on kafka so i think we can cross that bridge when we get there